common/utils.h

```diff
@@ -59,6 +59,7 @@ uint64_t read_eeprom_multi(uint32_t addr);
 #define EBUF_HARDFAULT      7
 #define EBUF_ASSERT         8
 #define EBUF_STACKOVERFLOW  9
+#define EBUF_VOLT_NORMAL    10 // FIXME it is now item 10, is it an issue?
```

Comment:
yes, this is an issue, you will need to put the value of EBUF_WITH_DATA and a higher number and reorganize all the other ones to be sequential. Those below EBUF_WITH_DATA are those where the error code is just done entry (no continuation codes) while the ones w/error codes greater than EBUF_WITH_DATA are those with continuation/multi-entry codes.  When the MCU codes are updated, the old error log entries will be invalid (no big deal)

Reply:
Done. (EBUF_WITH_DATA is set to 11 and the multi-entry codes are incremented by 1)